apps/meteor/server/settings/message.ts

```diff
@@ -385,4 +385,35 @@ export const createMessageSettings = () =>
 			type: 'boolean',
 			public: true,
 		});
+
+		await this.add('Message_CustomFields_Enabled', false, {
+			type: 'boolean',
+		});
+		await this.add(
+			'Message_CustomFields',
+			`
+{
```

Comment:
I'm not sure if it's possible, but can we validate this to ensure it's an object?

Reply:
It's not possible. It validates if it's a valid JSON only